src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/InstancesController.cs

```diff
@@ -296,18 +292,57 @@ public class InstancesController : ControllerBase
             return Created(url, instance);
         }
 
+        /// <summary>
+        /// Add complete confirmation.
+        /// </summary>
+        /// <remarks>
+        /// Add to an instance that a given stakeholder considers the instance as no longer needed by them. The stakeholder has
+        /// collected all the data and information they needed from the instance and expect no additional data to be added to it.
+        /// The body of the request isn't used for anything despite this being a POST operation.
+        /// </remarks>
+        /// <param name="org">unique identifier of the organisation responsible for the app</param>
+        /// <param name="app">application identifier which is unique within an organisation</param>
+        /// <param name="instanceOwnerPartyId">The party id of the instance owner.</param>
+        /// <param name="instanceGuid">The id of the instance to confirm as complete.</param>
+        /// <returns>Returns a list of the process events.</returns>
+        [Authorize(Policy = AuthzConstants.POLICY_INSTANCE_COMPLETE)]
+        [HttpPost("{instanceOwnerPartyId:int}/{instanceGuid:guid}/complete")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [Produces("application/json")]
+        public async Task<ActionResult<Instance>> AddCompleteConfirmation(
+            [FromRoute] string org,
+            [FromRoute] string app,
+            [FromRoute] int instanceOwnerPartyId,
+            [FromRoute] Guid instanceGuid)
+        {
+            try
+            {
+                Instance instance = await _instanceService.AddCompleteConfirmation(instanceOwnerPartyId, instanceGuid);
+                SelfLinkHelper.SetInstanceAppSelfLinks(instance, Request);
+
+                return Ok(instance);
+            }
+            catch (PlatformHttpException platformHttpException)
+            {
+                string message = $"Adding complete confirmation to instance {instanceOwnerPartyId}/{instanceGuid} failed";
+                return HandlePlatformHttpException(platformHttpException, message);
+            }
+            catch (Exception exception)
+            {
+                return ExceptionResponse(exception, $"Adding complete confirmation to instance {instanceOwnerPartyId}/{instanceGuid} failed");
+            }
```

Comment:
Could we consider simplifying this as the exceptionResponse function also handles PlatformHttpExceptions? We could simply call HandlePlatfformHttpException from the ExceptionResponse method.

Reply:
I combined HandlePlatformHttpException and ExceptionResponse into ExceptionResponse and removed all instances of catch PlatformHttpException.